bigtable/google/cloud/bigtable/row_set.py

```diff
@@ -29,6 +29,32 @@ def __init__(self):
         self.row_keys = []
         self.row_ranges = []
 
+    def __eq__(self, other):
+        if not isinstance(other, self.__class__):
+            return NotImplemented
+
+        if len(other.row_keys) != len(self.row_keys):
+            return False
+
+        if len(other.row_ranges) != len(self.row_ranges):
+            return False
+
+        if not sorted(other.row_keys) == sorted(self.row_keys):
+            return False
```

Comment:
Sorting might be a little expensive.  Can we achieve the same goal by creating sets?  E.g.:

```python
        if not set(other.row_keys) == set(self.row_keys):
            return False
```

Reply:
Yes, we can use the set here. Updated the code.